src/reducers/channelStores/verusid.js

```diff
@@ -29,13 +32,19 @@ export const channelStore_verusid = (state = {
     case SIGN_OUT_COMPLETE:
       return {
         openCoinChannels: {},
-        watchedVerusIds: {}
+        watchedVerusIds: {},
+        pendingIds: {}
       }
     case SET_WATCHED_VERUSIDS:
       return {
         ...state,
         watchedVerusIds: action.payload.watchedVerusIds
       }
+    case SET_VERUSID_NOTIFICATIONS:
```

Comment:
As mentioned in another comment, this is the bit I advise to change to SET_PENDING_VERUSIDS